flow-typed/npm/@mapbox/react-native-mapbox-gl_vx.x.x.js

```diff
@@ -0,0 +1,263 @@
+// flow-typed signature: f658ad0d9bed2e875e2884e6ed9fd396
```

Comment:
I removed all unnecessary stubs some time ago. I'm quite sure these stubs are not required. Therefore I'd say we remove them for now.